src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body {
+  background-color: whitesmoke;
+}
+
+button {
+  padding: 20px;
+  cursor: pointer;
+  border: 0;
+  margin: 0;
+}
+
+.list-container {
+  background: white;
+  box-shadow: 0 2px 5px lightgray;
+  width: 500px;
+  margin: 100px auto;
+}
+
+/* Title */
+
+.list-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 10px;
+  border-bottom: 1px solid lightgray;
+}
```

Comment:
This could be used as a different class as you use this snippet in two different classes.
``` 
display: flex;
justify-content: space-between;
align-items: center;
```